mongooseCrud/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  url: `mongodb://127.0.0.1:27017/ajay`
```

Comment:
db name should not be in URL

Reply:
sir mene url mongooseConnection mw hi likhi he